gcloud/gcloud_kms.go

```diff
@@ -192,6 +187,82 @@ func (g *gcloudKmsSecrets) PutSecret(
 	)
 }
 
+func getRSALimit(pk *rsa.PublicKey, hash *hash.Hash) int {
+	return pk.Size() - 2*(*hash).Size() - 2
```

Comment:
add a doc here on how this RSA limit is calculated. Why is the last 2 being subtracted? Is that for padding?

Reply:
updated